sigal/filters.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding:utf-8 -*-
+
+# Copyright (c) 2016 - Alexandre Chataignon
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+from __future__ import absolute_import
+import datetime
+import locale
+
+filters = [] # Filters to be exported
+
+def date_reformat(value, format_to, format_from='%Y-%m-%d'):
+    return datetime.datetime.strptime(value, format_from).strftime(format_to).decode(locale.getpreferredencoding())
```

Comment:
What is the goal of this filter ? There are indeed some hard-coded date formats in sigal (for exif date I think), so it may be easier to add a setting to control the date format instead of having to convert it later. 
Also the `decode` call is probably needed only for Python2 ?